test/server/badge.routes.test.js

```diff
@@ -66,7 +66,7 @@ describe("badge.routes.test.js", () => {
       expect(res.status).toEqual(200);
     }, timeout);
 
-    test("loads the banner (png)", async () => {
+    test.skip("loads the banner (png)", async () => {
```

Comment:
why skipping this?

Reply:
It's relying on [convert-svg-to-png](https://www.npmjs.com/package/convert-svg-to-png), relying itself on [Chrome/Puppeter](https://github.com/GoogleChrome/puppeteer), which is currently broken in the Circle CI 2 environment.

That is fixable, we just have to figure the missing Linux libraries necessary for Chrome to run.